chaos-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: litmus
+    app.kubernetes.io/version: v2.0.0
+    app.kubernetes.io/component: operator
+    app.kubernetes.io/part-of: litmus
+    app.kubernetes.io/managed-by: kubectl
```

Comment:
this label should not be populated, if need be then should be populated to helm `.release.service`, this label is used by helm to identify the resources managed by it , please review all labels based on https://helm.sh/docs/chart_best_practices/labels/#standard-labels